src/logic/caculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
```

Comment:
- [ ] Great job so far, unfortunately, I have trouble running your application and I see the result in the browser even after running `npm install and npm start`, instead I'm getting this error message 👇🏻 Kindly fix this 🥲 

- Console:
![errormessage](https://user-images.githubusercontent.com/79058364/173178253-c3c83367-fb8b-40c9-973e-672fb371518a.png)

- Brower:

![messbrow](https://user-images.githubusercontent.com/79058364/173178288-47b3f503-ef41-43a4-84c8-6ea23ddb6980.png)